packages/action-menu/test/action-menu-groups.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { elementUpdated, expect, oneEvent } from '@open-wc/testing';
+import { fixture } from '../../../test/testing-helpers.js';
+import { sendKeys } from '@web/test-runner-commands';
+
+import { groupsWithSelects } from '../stories/action-menu.stories.js';
+import { ActionMenu } from '@spectrum-web-components/action-menu';
+import { MenuItem } from '@spectrum-web-components/menu';
+
+describe('Action Menu - Groups', () => {
+    it('throws focus into the Menu when opened', async function () {
+        const el = await fixture<ActionMenu>(
+            groupsWithSelects({ onChange: () => {} })
+        );
+
+        const firstGroup = el.querySelector('sp-menu-group') as HTMLElement;
+        const firstItem = el.querySelector('sp-menu-item') as MenuItem;
+
+        expect(firstItem.focused).to.be.false;
+        expect(document.activeElement === firstGroup).to.be.false;
+
+        const opened = oneEvent(el, 'sp-opened');
+        el.focus();
+        await sendKeys({
+            press: 'ArrowDown',
+        });
+        await opened;
+
+        expect(firstItem.focused).to.be.true;
+        expect(
+            document.activeElement === firstGroup,
+            document.activeElement?.localName
+        ).to.be.true;
+    });
+
+    it.only('toggles child group with `selects="multiple"`', async function () {
```

Comment:
Drive by comment noting the test above this one being skipped

Reply:
😱 Thanks!